### PR TITLE
Added acceptFirstBaseline option.

### PR DESCRIPTION
**Pull request by tricoder42**
Test should fail when baseline doesn't exist in some environments (like CI). I've added simple option to mugshot constructor, whether test should fail or pass when baseline doesn't exist.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ The options object is optional and has the following keys:
   interface](lib/interfaces/png-processor.js),
 - **rootDirectory** - the path where the screenshots and diffs will be saved;
   defaults to `visual-tests`.
+- **acceptFirstBaseline** - whether the test should pass if the selector is
+  tested for the first time. Defaults to `true`.
```
Comment (andrei-picus-hs) [inline README.md]: Also add: `The baseline will be saved to the file system regardless of this option`.

**Comment by andrei-picus-hs** [comment]
@tricoder42 the feature seems legit. The PR needs a bit more work:

- [ ] the new param needs to be documented in the [docstring](https://github.com/uberVU/mugshot/blob/9ee93329c5b6fe07f9e1c8cbdc31c117183381eb/lib/mugshot.js#L75-L83),
- [ ] a [unit test](https://github.com/uberVU/mugshot/blob/9ee93329c5b6fe07f9e1c8cbdc31c117183381eb/tests/unit/no-baseline.js) needs to be written and
- [ ] an [integration test](https://github.com/uberVU/mugshot/blob/9ee93329c5b6fe07f9e1c8cbdc31c117183381eb/tests/integration/test.js) needs to be written.

**Comment by tricoder42** [comment]
PR updated. Not sure about the integration test though. I can't change mugshot options inside the first test suite, so I've created a new one.

**Comment by andrei-picus-hs** [comment]
@tricoder42 thanks!

**Pull request merged**